app/page.tsx

```diff
@@ -1,77 +1,5 @@
-import Link from 'next/link';
-import Image from 'next/image';
+import { redirect } from 'next/navigation';
 
 export default async function Index() {
-  return (
-    <>
-      <header className=' flex justify-center bg-background py-2' role='banner'>
-        <Image
-          src='/KINDLY_LOGO.png'
-          alt='Kindly clothes donations'
-          height={108}
-          width={329}
-        />
-      </header>
-      <main
-        className='grid items-center justify-items-center gap-5 p-4'
-        role='main'
-      >
-        <h1 className='italic'>A place to support Ukrainian refugees</h1>
-        <div className='flex'>
-          <p className='font-extrabold'>I would like to donate items</p>
-          <svg
-            width='88'
-            height='130'
-            viewBox='0 0 88 130'
-            fill='none'
-            xmlns='http://www.w3.org/2000/svg'
-          >
-            <path
-              d='M2 11.4408C115.392 -13.8709 102.294 17.8254 27 105.441'
-              stroke='#BBE3C3'
-              strokeWidth='5'
-            />
-            <path
-              d='M10.8446 115.424L22.0551 87.2624L40.8385 111.052L10.8446 115.424Z'
-              fill='#BBE3C3'
-            />
-          </svg>
-        </div>
-        <Link
-          href='/signup'
-          className='text-3xl font-extrabold italic text-primaryOrange'
-        >
-          Register here
-        </Link>
-        <div>
-          <svg
-            width='115'
-            height='165'
-            viewBox='0 0 115 165'
-            fill='none'
-            xmlns='http://www.w3.org/2000/svg'
-          >
-            <path
-              d='M85.5 160.5C-33.984 170.766 -17.7625 132.009 96 17.5'
-              stroke='#BBE3C3'
-              strokeWidth='5'
-            />
-            <path
-              d='M103.81 14.416L92.1209 42.6195L73.4517 17.6502L103.81 14.416Z'
-              fill='#BBE3C3'
-            />
-          </svg>
-          <p className='text-center font-extrabold'>
-            I’m a Ukrainian refugee and I would like to receive or donate items
-          </p>
-        </div>
-        <div className='flex space-x-2'>
-          <p>Already have an account?</p>
-          <Link href='/login' className='font-extrabold text-primaryOrange'>
-            Log in
-          </Link>
-        </div>
-      </main>
-    </>
-  );
+  redirect('/home-page');
```

Comment:
I think it would probably make sense to use this opportunity to correct a bit of a weird design decision that was made at the start of the project.

Since people will be starting from the home-page route we should probably just make that the actual home and move it to here.

This would mean correcting a few redirects throughout the codebase but now is probably the perfect opportunity to do it.

So that would leave us without a `/home-page` route and its contents would live in `/` instead. All others stay the same.

Reply:
Considering the change that had to be done, at first I considered:
1. Clearing up everything inside `/`.
2. Move the content of `/home-page` into `/`.

This is obviously consistent with the current state in `dev`, but it got me thinking that:
- The entire content of `/` will be lost.
- Backtracing `git` changes made on `/home-page` is now more difficult after the code gets moved around.
- If in the future there's the need to update `/` again, there will be another huge set of changes to overwrite it.

Making `/` a "pointer" would be a fix for that:
- Simpler: Changing the landing page would be as simple as changing the `redirect` target (no more rewriting entire files).
- Easier to trace: A page that is no longer `/` can still be kept in the codebase and used elsewhere.
- More flexible: `/` can be easily adapted in the future, if there's ever the need to redirect to different pages taking into account different criteria/logic.

However I understand that navigating to `/` and ending up in `/home-page` might be problematic on its own, so I'm going to rework this to move everything in `/home-page` to `/`.

Reply:
> This is obviously consistent with the current state in dev, but it got me thinking that:
>
> The entire (current) content of / will be lost.
> Backtracing git changes made on /home-page is now more difficult after the code gets moved around.
> If in the future there's the need to update / again, there will be another huge set of changes to overwrite it.

I did mull over the same but thinking that the current structure was really just a bad design to begin with (that page was pretty much a copy of out login page with a big logo on it) it seems like a bigger win to end up with a clearer navigation than safeguarding the history of that route :/